cmd/devtool/devtool.go

```diff
@@ -352,18 +349,29 @@ func createRunScript(ethAcctAddr, dataDir, serviceHost string, isBroadcaster boo
 
 	if !isBroadcaster {
 		script += fmt.Sprintf(` -initializeRound=true \
-    -serviceAddr %s:%d  -transcoder=true -orchestrator=true \
+    -serviceAddr %s:%d -orchestrator=true \
     -orchSecret secre -pricePerUnit 1
     `, serviceHost, mediaPort)
+		fName := fmt.Sprintf("run_%s_standalone_%s.sh", getNodeType(isBroadcaster), ethAcctAddr)
```

Comment:
The `standalone` script in your PR does not work, because it misses `\` before the last line.

You have the following:
```
    -orchSecret secre -pricePerUnit 1
     -transcoder=true
```

But it should be
```
    -orchSecret secre -pricePerUnit 1 \
     -transcoder=true
```

Reply:
Good spot, fixed